examples/integration/test/fixtures/bwb.xcodeproj/project.pbxproj

```diff
@@ -7325,6 +7333,28 @@
 				"BAZEL_COMPILE_TARGET_ID[sdk=iphoneos*]" = "//iOSApp/Source:iOSApp.library ios-arm64-min15.0-applebin_ios-ios_arm64-dbg-STABLE-4";
 				"BAZEL_COMPILE_TARGET_ID[sdk=iphonesimulator*]" = "$(BAZEL_COMPILE_TARGET_ID)";
 				BAZEL_LABEL = "@//iOSApp/Source:iOSApp";
+				BAZEL_OUTPUTS_DSYM = (
+					"\"bazel-out/applebin_ios-ios_x86_64-dbg-STABLE-7/bin/iOSApp/Source/CoreUtilsObjC/CoreUtilsObjC.framework.dSYM\"",
+					"\"bazel-out/applebin_ios-ios_x86_64-dbg-STABLE-7/bin/Lib/LibFramework.iOS.framework.dSYM\"",
+					"\"bazel-out/applebin_ios-ios_x86_64-dbg-STABLE-7/bin/UI/UIFramework.iOS.framework.dSYM\"",
+					"\"bazel-out/applebin_ios-ios_x86_64-dbg-STABLE-7/bin/WidgetExtension/WidgetExtension.appex.dSYM\"",
+					"\"bazel-out/applebin_ios-ios_x86_64-dbg-STABLE-7/bin/AppClip/AppClip.app.dSYM\"",
+					"\"bazel-out/applebin_watchos-watchos_x86_64-dbg-STABLE-9/bin/Lib/LibFramework.watchOS.framework.dSYM\"",
```

Comment:
Yes, it should use Spotlight to find the, so we should be able to copy it to sub-directory per-target.

I don't think it's a good idea to move it to the profile  pre-action, there may be other reasons for having dSYM's.

Alternatively, we can copy the dSYM for the current product only. How about this approach?

Reply:
> Alternatively, we can copy the dSYM for the current product only. How about this approach?

We would be missing transitive dSYMs for app extensions that aren't added to the scheme. So we need them to be transitive.

> I don't think it's a good idea to move it to the profile pre-action, there may be other reasons for having dSYM's.

What about the launch pre-action as well? And what other reasons are there for the rules_xcodeproj project to copy over dSYMs? lldb doesn't use them, and you can't submit them to the App Store.

Reply:
> What about the launch pre-action as well?
That will be fine. Or only for the launch target also should be fine. If would prefer only for the launch instead of pre-actions. Which one do you think is better?

> Though, what other reasons are there for the rules_xcodeproj project to copy over dSYMs?
Maybe for crash symbolization and for Crashlytics upload 🤔

Reply:
> Maybe for crash symbolization and for Crashlytics upload

You shouldn't use rules_xcodeproj built things for any submissions. We reserve the right to make slight changes for better debugging and/or integration: https://github.com/buildbuddy-io/rules_xcodeproj/blob/main/docs/faq.md#does-the-archive-action-work

> Or only for the launch target also should be fine.

Can't be just the Launch pre-action script, as that isn't run when Profiling. I actually think this should be a Profile pre-action script, because I don't think there are any other use cases outside of profiling, and that reduces the amount of copying for people that aren't profiling. You can add a script that does this to this array: https://github.com/buildbuddy-io/rules_xcodeproj/blob/9136dcb1dabe8e706d17a1cb9af33177617810d2/tools/generator/src/Extensions/XCScheme%2BExtensions.swift#L108.